### PR TITLE
Linked BE to FE in Voting Screen

### PR DESCRIPTION
**Pull request by kshahzada**
- Add card pre-loading and fixed the transition feature (you can't update state while it's swiping)
- Linked loaded data to backend
- Added voting functionality

### CODE REVIEW

**Comment by kshahzada** [comment]
The API calls should probably be refactored after we figure out the structure we want to use. I just tossed them into the models classes for the time being.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+.DS_Store
+.DS_Store
```
Comment (mhwatson) [inline .gitignore]: Duplicate. Note that there are no functional implications of this, but it's weird.

File: client/src/components/UI/cardQueue.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
```
Comment (mhwatson) [inline client/src/components/UI/cardQueue.jsx]: General comment:
This isn't really a UI component at all. It should be in the parent folder, as a standalone.

File: client/src/components/containers/Vote/index.jsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import DogCompare from '../../../components/dogCompare';
 
-import './vote.css';
+// import './vote.css';
```
Comment (mhwatson) [inline client/src/components/containers/Vote/index.jsx]: Looks like this was deleted. If that's the case, just remove the reference!

File: client/src/data/doggo-names.js [hunk 4]
```diff
@@ -38,6 +38,6 @@ export const doggos = [
 ];
 
 // Function that returns random doggo names
-const getDoggoName = () => doggos[Math.floor(Math.random() * (doggos.length + 1))];
+const getDoggoName = () => doggos[Math.floor(Math.random() * (doggos.length))];
```
Comment (mhwatson) [inline client/src/data/doggo-names.js]: Clearly I don't understand 0 based indexing :(

File: client/src/models/DogImage.js [hunk 5]
```diff
@@ -1,43 +1,27 @@
-/* global fetch */
+const axios = require('axios');
+const { getDoggoName } = require('../data/doggo-names');
```
Comment (mhwatson) [inline client/src/models/DogImage.js]: Nitpick: if we're using ES6 imports elsewhere we should be consistent

File: client/src/data/doggo-names.js [hunk 6]
```diff
@@ -38,6 +38,6 @@ export const doggos = [
...
 
-export default getDoggoName;
```
Comment (mhwatson) [inline client/src/data/doggo-names.js]: Think this is an AirBnB style guide thing...

File: client/src/components/containers/Vote/index.jsx (hunk 3 above)
Comment (kshahzada) [reply client/src/components/containers/Vote/index.jsx]: Don't know what the fix you want here is?

File: .gitignore (hunk 1 above)
Comment (kshahzada) [reply .gitignore]: Fixed - also added wildcard ignore for this.

File: client/src/components/UI/cardQueue.jsx (hunk 2 above)
Comment (kshahzada) [reply client/src/components/UI/cardQueue.jsx]: Moved.

File: client/src/components/containers/Vote/index.jsx (hunk 3 above)
Comment (kshahzada) [reply client/src/components/containers/Vote/index.jsx]: Deleted you schmuck

File: client/src/data/doggo-names.js (hunk 4 above)
Comment (kshahzada) [reply client/src/data/doggo-names.js]: Yup

File: client/src/models/DogImage.js (hunk 5 above)
Comment (kshahzada) [reply client/src/models/DogImage.js]: Changed.

File: client/src/data/doggo-names.js (hunk 6 above)
Comment (kshahzada) [reply client/src/data/doggo-names.js]: Changed - I just don't like to default anything... leaves room for ambiguity and naming inconsistencies... but I yield to AirBnB.

**Pull request merged**